### DISCUSSION TITLE
Hide REPL input box

### DISCUSSION THREAD

**Discussion by mafshin** [jupyterlite/jupyterlite#1277] [Q&A]
Is there any workaround to hide the REPL input at the bottom when running in a iframe? (I have the jupyterlab on a separate domain so accessing contents of the iframe is a cross-domain vilolation)

<img width="1287" alt="image" src="https://github.com/jupyterlite/jupyterlite/assets/4930000/cad86298-5f35-4d45-86af-2d05ddd8ffd5">

**Comment by jtpio** [comment]
@mafshin you mean for preventing users to execute code via the REPL?
